website/docs/r/scc_account_settings.html.markdown

```diff
@@ -1,34 +1,56 @@
 ---
 layout: "ibm"
-subcategory: "Security and Compliance Center"
 page_title: "IBM : ibm_scc_account_settings"
 description: |-
   Manages the account settings scc_account_settings
+subcategory: "SCC Admin Service API"
```

Comment:
here subcategory is changed to "SCC Admin Service API" from Security and Compliance Center and in the above docs you used "Admin Service API"

Please use the same subcateogory Security and Compliance Center to group all scc releated resource/datasource